frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap

```diff
@@ -12,6 +12,10 @@ Array [
     receiver.test
   </a>,
   ": ed25519:BgXFiJS...",
+  <p>
+    with permission undefined
+     and nounce undefined
```

Comment:
This is either a bad test or bad implementation. Please, make sure we don't render/test undefined.

Reply:
yup, I will update the test